src/pages/index.js

```diff
@@ -1,325 +1,253 @@
-                className='card'
-                href={link}
-                target='_blank'
-                rel='noreferrer noopener'
-                key={name}
-              >
-                <img src={`${baseUrl}img/${image}`} alt={name} />
-              </a>
-            )
-          }
-        })}
-      </div>
-    )
-
-    const Wallets = () => (
-      <CardBlock id='wallets'>
-        <h2>Web Monetization Wallets</h2>
-        <p>These providers offer ILP-enabled wallets.</p>
-        <CardSet cards={siteConfig.customFields.wallets} />
-        <MergeCard label='wallet' />
-      </CardBlock>
-    )
-
-    const Providers = () => (
-      <CardBlock id='providers'>
-        <h2>Web Monetization Providers</h2>
-        <p>These providers offer Web Monetization services</p>
-        <CardSet cards={siteConfig.customFields.providers} />
-        <MergeCard label='providers' />
-      </CardBlock>
-    )
+        )
+      }
+    })
+  }
 
-    const Browsers = () => (
-      <CardBlock id='browsers'>
-        <h2>Web Monetization Supported Browsers</h2>
+  const PrCta = (props) => {
+    return (
+      <div className='home-cta'>
+        <img className='icon--merge' src='/img/icon-merge.svg' alt='' />
         <p>
-          These browsers implement Web Monetization natively or via an extension
+          Do you know another {props.label}?{' '}
+          <a
+            href='https://github.com/WICG/webmonetization/edit/main/docusaurus.config.js'
+            target='_blank'
+            rel='noreferrer noopener'
+          >
+            Make a PR
+          </a>
         </p>
-        <CardSet cards={siteConfig.customFields.browsers} />
-        <MergeCard label='supported browser' />
-      </CardBlock>
-    )
-
-    const Search = () => (
-      <CardBlock id='search'>
-        <h2>Web Monetization Enabled Search Engines</h2>
-        <p>These search engines implement Web Monetization features</p>
-        <CardSet cards={siteConfig.customFields.search} />
-        <MergeCard label='search engine' />
-      </CardBlock>
-    )
-
-    const Tools = () => (
-      <CardBlock id='tools'>
-        <h2>Web Monetization Tools</h2>
-        <p>These tools enable Web Monetization features</p>
-        <CardSet cards={siteConfig.customFields.tools} />
-        <MergeCard label='tool' />
-      </CardBlock>
+      </div>
     )
+  }
 
-    const Resources = () => (
-      <Container padding={['left', 'right']} className='resources'>
-        <h2>Resources</h2>
-        <div className='resourceContainer'>
-          <div className='resource'>
-            <h3>Documentation</h3>
-            <p>
-              Resource on how to add Web Monetization to your site.
-              <br />
-              <a href={docUrl('docs/getting-started', this.props.language)}>
-                Read docs ›
-              </a>
-            </p>
-          </div>
-          <div className='resource'>
-            <h3>Explainer</h3>
+  return (
+    <Layout
```

Comment:
Markup has been greatly simplified to reduce redundant wrappers.
Due to the design of the site's hero section with a full bleed background, `Container` is used as an inner wrapper instead to keep the content within a specific max-width. The max-width values out-of-the-box from Material UI are fairly reasonable and suit the type of content we have, so that's what we're using now.